feel/src/main/scala/io/camunda/zeebe/feel/impl/FeelToMessagePackTransformer.scala

```diff
@@ -63,6 +63,13 @@ class FeelToMessagePackTransformer {
           }
         }
       }
+      case ValTime(time) => writeValue(ValString(time.format))
```

Comment:
:wrench: We could extract a function to write a string value.

Instead of creating a new object `ValString`, we could extract a function (e.g. `writeStringValue(time.format)`). However, the difference is not significant :sweat_smile: